docs/source/cli/graph-construction/distributed/gsprocessing/input-configuration.rst

```diff
@@ -489,9 +489,174 @@ arguments.
         You can find all models in the `Huggingface model repository <https://huggingface.co/models>`_.
       - ``max_seq_length`` (Integer, required): Specifies the maximum number of tokens of the input.
         You can use a length greater than the dataset's longest sentence; or for a safe value choose 128. Make sure to check
-        the model's max suported length when setting this value,
+        the model's max supported length when setting this value.
 
---------------
+
+..  _gsprocessing-multitask-ref:
+
+Creating a graph for multi-task training
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To create a graph for multi-task training, you simply need to
+define a label in your config for each of the labels you want to
+use during training. GSProcessing will generate separate
+train/val/test masks for each of your labels, which
+will be named ``<mask_type>_<label_column>``, e.g.
+if your label columns were ``label_class``, and ``label_reg``
+GSProcessing will generate masks named as ``train_mask_label_class``,
+``train_mask_label_reg`` etc. For link prediction tasks, the masks
+will have ``_lp`` added as a suffix, e.g. ``train_mask_lp``.
+
+After partitioning the data, you can then provide these masks
+in your train YAML file to use during multi-task training.
+
+For details on running multi-task training see
+:doc:`/advanced/multi-task-learning`.
+
+Here we list an example multi-task GSProcessing config for the ACM data described in
+:doc:`/tutorials/use-own-data`, where we prepare a node classification label for the
+``paper`` node type, and a link prediction task on the ``paper,citing,paper`` edge
+type.
+
+.. raw:: html
+
+   <details>
+   <summary><a>Example multi-task GSProcessing config</a></summary>
```

Comment:
This tag (and corresponding closing tag below) create collapsible code sections see the rendered example:

<img width="746" alt="image" src="https://github.com/user-attachments/assets/142a2c11-6f75-4337-8c74-3802c4575518">